substrate/frame/revive/rpc/Cargo.toml

```diff
@@ -65,6 +77,7 @@ tokio = { workspace = true, features = ["full"] }
 
 [features]
 example = ["rlp", "subxt-signer"]
+sqlite = []
```

Comment:
Why is it a feature and not a runtime decision which database to use? Due to feature unification it is not a good idea to make this a feature.

Reply:
Removed in last commit